prime-router/src/test/kotlin/fhirengine/azure/FHIRRouterIntegrationTests.kt

```diff
@@ -645,11 +647,16 @@ class FHIRRouterIntegrationTests : Logging {
                     )
                 }
 
-            assertThat(reportAndBundles).transform { pairs ->
-                pairs.map {
-                    it.second.toString(Charsets.UTF_8)
-                }
-            }.containsOnly(File(FILTERED_OBSERVATIONS_FHIR_URL).readText())
+            var fhirBundlesAsObjectsOnly = reportAndBundles.map { it.second.toString(Charsets.UTF_8) }
+                                                           .map { FhirTranscoder.decode(it) }
+
+            // there should only be one observation of five remaining, and the code of that observation
+            // should be 94558-5
+            for (fhirBundleAsObject in fhirBundlesAsObjectsOnly) {
+                assertThat(fhirBundleAsObject.getObservations().size == 1)
```

Comment:
I think the preference would be to extract out the string of what we expect the FHIR bundle to be and do a string comparison.  This way we can make sure that we're not accidentally filtering anything else out of the bundle